### PR TITLE
added script to run tests and related commands

### PR DESCRIPTION
**Pull request by ajwann**
If everything looks good with this test script, I can add it to the other lucky libraries.

### CODE REVIEW

File: bin/test [hunk 1]
```diff
@@ -0,0 +1,8 @@
+#! /bin/bash
+
+printf "\nRunning 'crystal spec'\n\n"
+crystal spec
+printf "\nRunning 'shards build'\n\n"
+shards build
+printf "\nRunning 'crystal tool format src'\n\n"
+crystal tool format src
```
Comment (paulcsmith) [inline bin/test]: Could you add “spec” after  “src” so it formats both directories?

File: bin/test [hunk 2]
```diff
@@ -0,0 +1,8 @@
+#! /bin/bash
+
+printf "\nRunning 'crystal spec'\n\n"
+crystal spec
+printf "\nRunning 'shards build'\n\n"
+shards build
+printf "\nRunning 'crystal tool format src'\n\n"
```
Comment (ajwann) [inline bin/test]: I'm not sure if we want to run ```crystal tool format``` on more than the source directory. I did see some errors when I ran it on the project root.

Comment (paulcsmith) [inline bin/test]: Maybe “formatting code with crystal tool format src spec”

File: bin/test (hunk 1 above)
Comment (ajwann) [reply bin/test]: Sure, no problem.

**Comment by ajwann** [comment]
Actually let me add one more step to the contributing section, it looks a bit odd the way I left it.

**Pull request merged**